src/components/v5/common/CompletedAction/partials/StreamingPayment/partials/StreamingPaymentWidget/StreamingPaymentWidget.tsx

```diff
@@ -0,0 +1,304 @@
+
+  const checkIfHasEnoughFunds = useHasEnoughTokensToClaim(
+    streamingPayment?.nativeDomainId || Id.RootDomain,
+    streamingPayment?.tokenAddress || '',
+    amounts.amountAvailableToClaim,
+  );
+
+  const [hasEnoughFunds, setHasEnoughFunds] = useState(checkIfHasEnoughFunds());
+
+  const claim = useAsyncFunction({
+    submit: ActionTypes.STREAMING_PAYMENT_CLAIM,
+    error: ActionTypes.STREAMING_PAYMENT_CLAIM_ERROR,
+    success: ActionTypes.STREAMING_PAYMENT_CLAIM_SUCCESS,
+  });
+
+  const handleClaim = async () => {
+    if (!streamingPayment) {
+      return;
+    }
+
+    setIsLoading(true);
+
+    try {
+      const hasEnoughFundsToMakeClaim = checkIfHasEnoughFunds();
+
+      if (!hasEnoughFundsToMakeClaim) {
+        setHasEnoughFunds(false);
+        return;
+      }
+
+      const claimPayload: ClaimStreamingPaymentPayload = {
+        colonyAddress: colony.colonyAddress,
+        streamingPaymentsAddress: streamingPaymentsAddress ?? '',
+        streamingPayment,
+      };
+
+      await claim(claimPayload);
+
+      refetchStreamingPayment();
+      fetchCurrentBlockTime();
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  if (!streamingPayment) {
+    return null;
+  }
+
+  const { tokenAddress, createdAt } = streamingPayment;
+  const selectedToken = getSelectedToken(colony, tokenAddress || '');
+
+  return (
+    <div
+      className={clsx('flex flex-col', {
+        'gap-2': !action.isMotion,
+        'gap-8': action.isMotion,
+      })}
+    >
+      <MenuWithSections
+        sections={[
+          ...(hasEnoughFunds
+            ? []
+            : [
+                {
+                  key: '0',
+                  className:
+                    'bg-negative-100 text-negative-400 flex flex-col gap-1 items-start !py-3',
+                  content: (
+                    <>
+                      <p className="text-sm">
+                        {formatText(MSG.insufficientFunds)}
+                      </p>
+                      <button
+                        type="button"
+                        className="inline-block text-xs font-medium underline"
+                        onClick={() =>
+                          toggleActionSidebarOn({
+                            [ACTION_TYPE_FIELD_NAME]: Action.TransferFunds,
+                          })
+                        }
+                      >
+                        {formatText(MSG.transferFunds)}
+                      </button>
+                    </>
+                  ),
+                },
+              ]),
+          {
+            key: '1',
+            content: (
+              <>
+                <div className="flex items-center justify-between gap-4">
+                  <h4 className="text-1">{formatText(MSG.streamingStatus)}</h4>
+                  <LoadingSkeleton
+                    isLoading={isLoading}
+                    className="h-5 w-14 rounded"
```

Comment:
This should match the height of the pill to prevent this jumping:

https://github.com/user-attachments/assets/6b6d8f0f-f539-4d64-8c34-23e66d50aee8